tests/ReplayData/AuthenticatedUser.testEmails.txt

```diff
@@ -7,18 +7,18 @@ None
 None
 200
 [('status', '200 OK'), ('x-ratelimit-remaining', '4934'), ('content-length', '64'), ('server', 'nginx/1.0.13'), ('connection', 'keep-alive'), ('x-ratelimit-limit', '5000'), ('etag', '"ea6dacf29569317ccf460b4bb07075e5"'), ('date', 'Sun, 20 May 2012 12:41:39 GMT'), ('content-type', 'application/json; charset=utf-8')]
-["vincent@vincent-jacques.net","github.com@vincent-jacques.net"]
+[{"email": "vincent@vincent-jacques.net", "primary": True, "verified": True, "visibility": "private"},{"email": "github.com@vincent-jacques.net", "primary": False, "verified": True, "visibility": None}]
```

Comment:
I understand the failure now -- this is the Python representation of the data, but the test machinery expects JSON, so that data[0] error is masking a JSON decode error.

```
[{"email": "vincent@vincent-jacques.net", "primary": true, "verified": true, "visibility": "private"}, {"email": "github.com@vincent-jacques.net", "primary": false, "verified": true, "visibility": null}]
```

is the correct JSON representation that should go here -- and in the other replay data.

Reply:
OK, thanks, I will fix it.